src/settings.cpp

```diff
@@ -237,6 +236,25 @@ settings::settings(config::settings context)
         {
         }
     }
+    max_money_ = recursive_money_ * subsidy_interval_;
+}
+
+uint64_t settings::bitcoin_to_satoshi(uint64_t bitcoin_units) const
```

Comment:
For property methods I generally just use "value" for the parameter since the parameter name is redundant with the property name.

Reply:
Done.